stdlib/tools/swift-reflection-test/swift-reflection-test.c

```diff
@@ -34,6 +34,10 @@
 #include <fcntl.h>
 #endif
 
+#if defined(__clang__) || defined(__GNUC__)
+#define NORETURN __attribute__((noreturn))
+#endif
```

Comment:
Do we need an `#else #define NORETURN` here for other compilers?

Reply:
Oh, duh.  Ya. Normally I'd not ifdef this, but given that Windows support is being pushed forward, I thought I'd make a nod towards portability. See also: #26868